cabot/cabotapp/views.py

```diff
@@ -864,3 +834,20 @@ def _handle_action(self, check, action):
             return 'counter reset to 0'
 
         raise ViewError("invalid action '{}'".format(action), 400)
+
+
+class SendTestAlertView(LoginRequiredMixin, View):
+    def get(self, *args, **kwargs):
+        alert = AlertPlugin.objects.get(pk=kwargs['alert_pk'])
+
+        try:
+            alert.send_test_alert(self.request.user)
+            messages.success(self.request, "Alert sent.", extra_tags="alert-success")
+        except Exception as e:
+            logger.exception("Failed to send test alert to user {}: {}".format(self.request.user, e))
```

Comment:
nit: for exception logging in rollbar, use %s instead of format strings to properly group exceptions of the same type